lib/routes/_core_profile.js

```diff
@@ -84,6 +85,9 @@ module.exports = {
       if (typeof body.authenticatorAssuranceLevel !== 'undefined') {
         result.twoFactorAuthentication = body.authenticatorAssuranceLevel >= 2;
       }
+      if (typeof body.profileChangedAt !== 'undefined') {
+        result.profileChangedAt = body.profileChangedAt;
+      }
```

Comment:
I'll add here as an early comment, we should probably treat this new field as an internal implementation detail and avoid returning it to reliers as part of the user's profile data.  We don't expect them to have any use for it.